profiles/coreos/arm64/package.use

```diff
@@ -1,5 +1,7 @@
 # arm64 use
 
+net-dns/bind-tools -gssapi
```

Comment:
Maybe add a comment here pointing to the issue or at least a comment explaining that enabling gssapi causes build failures?

Reply:
Added comments as suggested